py/trtorch/csrc/utils.h

```diff
@@ -0,0 +1,138 @@
+#pragma once
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+
+#include <functional>
+#include <iostream>
+#include <string>
+#include "NvInfer.h"
+
+namespace tensorrt {

```

Comment:
put this in the `trtorch::pyapi::util` namespace. Also lets call it the `util` namespace instead of utils to be consistent with core

Reply:
Fixed it. This was inherited from TRT source code. I cleaned it up  to be as minimal and relevant as possible.